test/1_stdlib/StaticString.swift

```diff
@@ -15,7 +15,7 @@ StaticStringTestSuite.test("PointerRepresentation/ASCII/Empty") {
     utf8 in
     expectEqual(0, utf8.count)
   }
-  expectEqual("", String(str))
+  expectEqual("", str.description)
```

Comment:
That entails adding an initializer that could create `StaticString`s from `String`s, which I don't think is a goal of the API.